sdk/tables/azure-data-tables/test/ut/table_client_test.cpp

```diff
@@ -156,7 +156,6 @@ namespace Azure { namespace Data { namespace Test {
     EXPECT_EQ(response.Value.Logging.Delete, false);
     EXPECT_EQ(response.Value.HourMetrics.RetentionPolicyDefinition.IsEnabled, true);
     EXPECT_EQ(response.Value.HourMetrics.Version, "1.0");
-    EXPECT_EQ(response.Value.HourMetrics.IsEnabled, true);
```

Comment:
It's weird but had to do it because something changed in the infra and this thingy now is disabled in live , i'm activelly working on getting access to the TME subscription where i can deploy the resources to debug this. This causes the live tests to fail for tables, and have been failing for a while. until i can get this debugged, since i couldn't deploy the resource i also couldn't record the tests again, thus ...